src/script/conversation/ConversationFederationUtils.ts

```diff
@@ -59,3 +59,47 @@ export function processFederationDeleteEvent(
 
   return result;
 }
+
+export interface FederationConnectionRemovedResult {
+  conversationsToDeleteUsers: {conversation: Conversation; users: User[]}[];
+}
```

Comment:
suggestion: we could simplify the return type to be a simple array
```suggestion
export type FederationConnectionRemovedResult = {conversation: Conversation; users: User[]}[]
```